account_invoice_export/models/transmit_method.py

```
@@ -21,6 +21,14 @@ def get_transmission_http_header(self):
 
         """
         self.ensure_one()
-        auth = "{}:{}".format(self.destination_user, self.destination_pwd,)
+        auth = "{}:{}".format(self.destination_user or "", self.destination_pwd or "")
```

wouldn't be better to raise a specific error?

---

Well my thought was letting the call be made and let `request` or the distant server raise an error.
From what I know the password could be empty ?

---

yeah but the error might be not so clear for the user. I guess destination user is mandatory? If yes, raising a UserError here or before should be better. Unless I miss something else.

---

From what  I read online and what people understand from the  specs https://datatracker.ietf.org/doc/html/rfc1945#section-11 Basic Authentication allows for an empty user name and password. That is sending just the semi column is a valid option.
In that light the code is correct.